<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DobObjectEditWidget</class>
 <widget class="QWidget" name="DobObjectEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1438</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="100,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QTreeView" name="objectEditTreeView"/>
     </item>
     <item>
      <widget class="QScrollArea" name="filterScroller">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="filterArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>650</width>
          <height>36</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="filterAreaLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="nameFilterEdit">
           <property name="toolTip">
            <string>Accepts regular expression.</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>Filter</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="valueFilterEdit">
           <property name="toolTip">
            <string>Accepts regular expression.</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>Filter</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="nullFilterCombo">
           <property name="styleSheet">
            <string notr="true">text-align: left;</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Filter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="changedFilterCombo">
           <property name="styleSheet">
            <string notr="true">text-align: left;</string>
           </property>
           <property name="iconSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Filter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="typeFilterEdit">
           <property name="toolTip">
            <string>Accepts regular expression.</string>
           </property>
           <property name="placeholderText">
            <string>Filter</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="OperationsWidget" name="operationsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OperationsWidget</class>
   <extends>QWidget</extends>
   <header>operationswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
